### ISSUE TITLE
单卡80G 微调 OOM

### ISSUE THREAD

**Issue by Mors20** [ASLP-lab/OSUM#44]
你好，请问一下，单卡A800 80G，在微调OSUM, deepseed.initialize时报OOM

        model, optimizer, _, scheduler = deepspeed.initialize(
            args=args,
            model=model,
            optimizer=optimizer,
            lr_scheduler=scheduler,
            model_parameters=model.parameters())

Zero的配置用的是config/ds_stage2.json, 完全按照run_huawei_2p_master.sh原本的配置来的

请问这是为什么？非常感谢！
